src/main.rs

```diff
@@ -0,0 +1,44 @@
+#![cfg_attr(feature = "cargo-clippy", deny(clippy::expect_fun_call))]
+#![cfg_attr(
+    feature = "cargo-clippy",
+    warn(
+        clippy::result_unwrap_used,
+        clippy::panicking_unwrap,
+        clippy::option_unwrap_used
+    )
+)]
+#[macro_use]
+extern crate slog;
+extern crate clap;
+extern crate rbdate;
+extern crate slog_async;
+extern crate slog_term;
+
+#[macro_use]
+mod macros;
+mod configuration_parameters;
+mod init;
+mod log;
+mod process;
+mod read_data_files;
+mod report_generator;
+mod structs;
+
+use init::init_loggers;
+use process::parameter_calls;
+use std::time::SystemTime;
+use report_generator::write_summary;
+
+fn main() {
+    let start_aggregation_timer = SystemTime::now();
+
+    // initialize loggers
+    let app_name = "employee_report_generator";
+    println!("hello");
+    let (config_params, log, diagnostics_log) = init_loggers(app_name);
+    // process
+    parameter_calls(&config_params, &log, &diagnostics_log);
```

Comment:
I don't think this function is required

Reply:
it is throwing error if i remove this error getting log file